### ISSUE TITLE
Playback speed increased between two sessions

### ISSUE THREAD

**Issue by Kabouik** [Rudloff/viatcheslav#4]
Viatcheslav was working correctly earlier today, then I closed it and rebooted my computer, and now Viatcheslav is playing at like 200% speed with no sound (even for Live TV, it just waits for the stream for a few seconds and then plays the buffer at highspeed). Molotov still plays the same channels correctly. 

I deleted ~/.cache, updated Viatcheslav with `git clone` and ran `npm run linux-setup` again, but no success.

I have had this issue once with Molotov, and that was one reason why I wanted to use Viatcheslav instead, but I never identified the cause of the issue (and it disappeared somehow at a later point). I am sure I did something wrong, but can't pinpoint what. 

What is the player used by Viatcheslav under the hood? I'm running Linux and have mpv and gnome-mpv installed, no other players, but both play at the correct speed. I tried changing their playback speed with `[` and `]`, the default shortcuts, but it had no effect on Viatcheslav.

**Comment by Kabouik** [comment]
And it's back to normal now, I don't know what I have changed, weird.

**Comment by Rudloff** [comment]
Viatcheslav is just a wrapper and uses the same HTML5 player as molotov.tv so this is probably a Molotov bug.

**Comment by Kabouik** [comment]
But then I'm not sure how to interpret the fact that the Molotov application was working at correct speed when Viatcheslav was not. And a few months ago it was the opposite. Is there any way the wrappers can influence playback speed?

**Comment by Rudloff** [comment]
Playback speed can be set in JavaScript, but don't know if you triggered a bug in Molotov that incorrectly sets the speed or if you found some hidden keyboard shortcut to change the speed.

New time this happens, you can try to open the developer tools (*View > Toggle Developer Tools*) and type this in the JS console:
```js
document.querySelector('video').playbackRate = 1;
```

**Comment by Kabouik** [comment]
I've tried every key because I thought about a hidden keyboard shortcut too, but couldn't reproduce or revert the playback speed change.

I wonder if it was not related to a script I'm using to transfer the audio output to a third monitor (TV) though: https://hastebin.com/xowohifari Indeed, after I opened this issue, hitting Shift+F10 (my keyboard combination to execute this script) immediately caused the same issue in the Molotov application, but hitting it again had no effect. This is very weird though, as I use this script several times a day, often with Viatcheslav (or Molotov), and usually have no issue. I wonder if it can conflict with something else I still haven't identified and cause the issue just when all conditions are satisfied.

I'll make sure I check the JS console when the bug appears again and report the output here, thanks.
